### PR TITLE
chore: revise documentation

### PR DESCRIPTION
**Pull request by evenstensberg**
**What kind of change does this PR introduce?**

Documentation

**Did you add tests for your changes?**
N/A
**If relevant, did you update the documentation?**
Yea kinda 😄 

**Summary**
Revises some wording with a clearer langugage.

**Does this PR introduce a breaking change?**

No

**Other information**
N/A

### CODE REVIEW

**Review by alexander-akait — APPROVED** [review]
/cc @webpack/cli-team

File: README.md [hunk 1]
```diff
@@ -57,7 +59,7 @@ Thus, webpack CLI provides different commands for many common tasks.
 -   `build|bundle|b [entries...] [options]` - Run webpack (default command, can be omitted).
 -   [`configtest|t [config-path]`](./packages/configtest/README.md#webpack-cli-configtest) - Validate a webpack configuration.
 -   `help|h [command] [option]` - Display help for commands and options.
--   [`init|create|new|c|n [generation-path] [options]`](./INIT.md#webpack-cli-init) - Create a new webpack project.
+-   [`init|create|new|c|n [generation-path] [options]`](./packages/README.md#webpack-cli-init) - Create a new webpack project.
```
Comment (snitin315) [inline README.md:62]: The new link is invalid I think.

Comment (evenstensberg) [inline README.md:62]:
```suggestion
-   [`init|create|new|c|n [generation-path] [options]`](./packages/init/README.md#webpack-cli-init) - Create a new webpack project.
```

**Comment by evenstensberg** [comment]
PTAL

File: README.md [hunk 2]
```diff
@@ -57,7 +59,7 @@ Thus, webpack CLI provides different commands for many common tasks.
 -   `build|bundle|b [entries...] [options]` - Run webpack (default command, can be omitted).
 -   [`configtest|t [config-path]`](./packages/configtest/README.md#webpack-cli-configtest) - Validate a webpack configuration.
 -   `help|h [command] [option]` - Display help for commands and options.
--   [`init|create|new|c|n [generation-path] [options]`](./INIT.md#webpack-cli-init) - Create a new webpack project.
+-   [`init|create|new|c|n [generation-path] [options]`](./packages/init/README.md#webpack-cli-init) - Create a new webpack project.
```
Comment (snitin315) [inline README.md:62]: Init package was moved to generators. So the old link is good here.

Comment (evenstensberg) [reply README.md]: let's move the init readme to generators then. Following this logic, we would have had merge, bundle and every other command at root level too.

**Comment by evenstensberg** [comment]
PTAL

**Review by snitin315 — APPROVED** [review]

**Pull request merged**